ParallelPrimitives/RadixSort.cpp

```diff
@@ -131,9 +131,9 @@ int RadixSort::calculateWGsToExecute( oroDevice device ) noexcept
 	return props.multiProcessorCount * occupancy;
 }
 
-RadixSort::u32 RadixSort::configure( oroDevice device, const std::string& kernelPath, const std::string& includeDir ) noexcept
+RadixSort::u32 RadixSort::configure( oroDevice device, OrochiUtils* oroutils, const std::string& kernelPath, const std::string& includeDir ) noexcept
```

Comment:
Use reference instead of pointer since this cannot be null and yet we do not check for nullptr in the code.
```suggestion
RadixSort::u32 RadixSort::configure( oroDevice device, OrochiUtils& oroutils, const std::string& kernelPath, const std::string& includeDir ) noexcept
```